Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -711,15 +711,6 @@ protected void onPause() {
     protected void onDestroy() {
         if (sharedPref != null) {
             sharedPref.edit().putString(getResources().getString(in.juspay.mobility.app.R.string.ACTIVITY_STATUS), "onDestroy").apply();
-            String role = sharedPref.getString("ROLE_KEY", "null");
```

Comment:
is this change part of triggering waiting charges? also why is boradcast removed here, is it not being used?

Reply:
`ROLE_KEY` and `LOCATION_STATUS` we are not setting anywhere and not being used,
So it is never going to this If block, And it's redundant code, That's why I removed it